packages/lesswrong/server/editor/mentions-notify.ts

```diff
@@ -0,0 +1,46 @@
+import * as _ from 'underscore'
+import {createNotifications} from '../notificationCallbacksHelpers'
+import {notificationDocumentTypes} from '../../lib/notificationTypes'
+
+export interface PingbackDocumentPartial {
+  _id: string,
+  pingbacks?: {
+    Users: string[]
+  }
+}
+
+export const notify = async (currentUser: DbUser, collectionType: string, document: PingbackDocumentPartial, oldDocument?: PingbackDocumentPartial) => {
```

Comment:
Give this a name that more clearly says what it's for (i.e. notifyUsersAboutMentions)